plasmapy/utils/decorators/converter.py

```diff
@@ -0,0 +1,38 @@
+"""
+Decorator to convert units of functions in /physics methods
+"""
+from astropy import units as u
+import functools
+import inspect
+from plasmapy.utils.decorators import preserve_signature
+__all__ = ["from_radians_to_hz"]
+
+
+def from_radians_to_hz(fn):
```

Comment:
This decorator name is a little misleading. You're not converting radians to Hz, you're converting angular frequency to Hz.  I'd rename the decorator more along the lines of `rads_per_sec_to_hz` or `angular_freq_to_hz`.

Reply:
changing that on the next push